openmrs-19/src/main/java/org/motechproject/openmrs19/domain/OpenMRSPatient.java

```diff
@@ -34,23 +36,39 @@ public OpenMRSPatient(String patientId) {
      * @param facility  the facility by which the patient is treated
      */
     public OpenMRSPatient(String motechId, OpenMRSPerson person, OpenMRSFacility facility) {
-        this(null, motechId, person, facility);
+        this(motechId, person, facility, null);
     }
 
     /**
-     * Creates a patient with the given {@code motechId} and OpenMRS {@code patientId} based on the given {@code person}
+     * Creates a patient with the given {@code motechId} and {@code identifierTypesByValue} based on the given {@code person}
      * details and assigns it to the given {@code facility}.
      *
+     * @param motechId  the MOTECH ID of the patient
+     * @param person  the personal details about the patient
+     * @param facility  the facility by which the patient is treated
+     * @param identifierTypesByValue the identifiers of patient by value
+     */
+    public OpenMRSPatient(String motechId, OpenMRSPerson person, OpenMRSFacility facility, Map<String, String> identifierTypesByValue) {
+        this(null, motechId, person, facility, identifierTypesByValue);
```

Comment:
ByValue? What value? Can you just name it identifiers and explain in the javadoc what is the key and what is the value?